core/core-dotspacemacs.el

```diff
@@ -80,6 +80,9 @@ banner, `random' chooses a random text banner in `core/banners'
 directory. A string value must be a path to a .PNG file.
 If the value is nil then no banner is displayed.")
 
+(defvar dotspacemacs-version-check-enable t
```

Comment:
*I'm still not sure about the variable name, so I report my old comment here:*

Wouldn't `dotspacemacs-enable-version-check` or `dotspacemacs-version-check-enabled` be more correct? I'm just asking, I'm not English native :relaxed: